<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1">



<title></title>

<script src="libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-75471277-1', 'auto');
  ga('send', 'pageview');
</script>

<style type="text/css">code{white-space: pre;}</style>
<link rel="stylesheet"
      href="libs/highlight/default.css"
      type="text/css" />
<script src="libs/highlight/highlight.js"></script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<link rel="stylesheet" href="includes/main.css" type="text/css" />

</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>

<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script src="libs/navigation-1.0/tabsets.js"></script>
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->





<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <!-- NOTE: add "navbar-inverse" class for an alternate navbar background -->
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Computational Neurobiology Lab</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="research.html">Research</a></li>
        <li><a href="positions.html">Positions</a></li>
        <li><a href="members.html">Members</a></li>
        <li><a href="publications.html">Publications</a></li>
        <li><a href="toolsAndData.html">Tools and Data</a></li>
        <li><a href="contact.html">Contact</a></li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



</div>


<div id="research-interests" class="section level2">
<h2>Research Interests</h2>
<p>Broadly, we are interested in helping integrate epigenomic, transcriptomic, genetic, connectivity, imaging, clinical and epidemiological data related to the neurobiology of psychiatric disorders.</p>
<div id="polygenic-expression-mapping" class="section level3">
<h3>Polygenic expression mapping</h3>
<p>Over $400 million has been spent on the creation of comprehensive gene expression, connectivity and methylation atlases of the mouse and the human brain. In five years, more data will be available from the Allen Institute for Brain Science, the Human Connectome Project, the BRAIN Initiatives, and the European Human Brain Project. We will create methods and tools that will leverage this new data for fast analyses of neuroanatomy. Our first aim is to provide maps of where (in the brain) and when (during development) genes are expressed in the mouse and the human brain. Currently, tools for polygenic expression analysis are not available to scientists without advanced computational backgrounds. We seek to complement the large and expensive gene expression atlases with free online tools for polygenic mapping of the brain. Prototype versions of these tools are <a href="./toolsAndData.html">avaiable</a>. New versions will include tests for cell type specific patterns and expression datasets from the UK Brain Expression Consortium and the NIH Blueprint Non-Human Primate Atlas.</p>
<p>Decoding the genome has allowed us to discover associations between specific genes and diseases, as well as to measure expression of all genes in the genome in a specific tissue. Common mental illnesses are complex, caused by many genes possibly interacting with various environmental factors. Compounding this etiological complexity is the heterogeneity of gene expression in the brain, with over 75% of genes differentially expressed across brain regions or time. Many of the genes associated with mental illness have unknown roles. While many scientists are zooming in on these genes, they focus on specific brain regions at a specific age for a single gene. We use computational aproaches to complement targeted efforts with whole brain and genome perspectives. Using the prototype tools, we have discovered that genes previously associated with schizophrenia, autism, Parkinson’s disease, cardiovascular health, low early-life social class and human well-being are specifically expressed in brain regions of interest. We plan on extending these findings in the Neuroinformatics labratory.</p>
</div>
<div id="parkinsons-disease" class="section level3">
<h3>Parkinson’s disease</h3>
<p>We seek to test for interactions between Parkinson’s associated genes and the immune system in human gene expression and epigenetic resources. This interest builds on past work that revealed differential co-expression between the alpha-synuclein gene and interferon-gamma signalling genes.</p>
</div>
<div id="text-mining" class="section level3">
<h3>Text mining</h3>
<p>We are interested in using written text to better characterize mental health and addiction. This work is supported by the increasing amounts of our communications at that are archived online. Inspiration is provided by work that has examined novels for markers of congnitive decline <a href="http://llc.oxfordjournals.org/content/26/4/435.full">(Le, Lancashire, Hirst and Jokel, 2011)</a> and tweets of mothers for postpartum language changes <a href="http://research.microsoft.com/en-us/um/people/horvitz/predicting_postpartum_changes_chi_2013.pdf">(Choudhury, Counts and Horvitz, 2013)</a>.</p>
</div>
</div>
<div id="collaborators" class="section level2">
<h2>Collaborators</h2>
<p>Current and recent past collaborators include:</p>
<ul>
<li>Drs. Tomáš Paus and Zdenka Pausova in Toronto (my postdoctoral supervisors): population neuroscience, epigenetics and axon transport</li>
<li>Drs. Franziska Wagner and Rüdiger W Veh in Germany: gene expression analyses in the habenula</li>
<li>Dr. Noa Liscovitch in Israel: coexpression analyses of SNCA in Parkinson’s disease</li>
<li>Dr. Etienne Sibille in Toronto: neurobiology of depression and aging</li>
<li>Dr. Paul Pavlidis in Vancouver (my PhD supervisor): bioinformatics for neuroanatomical connectivity</li>
<li>Yohan Yee: text mining for Psychiatry</li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
